CONFIGURATION.md

```diff
@@ -61,7 +61,7 @@ The configuration file can supply the following properties, showing their defaul
 ### Configuration File Probe Settings ###
 
 Probes are the packages that `appoptics-apm` auto-instruments. Different types of probes have different
-configuration options. See `lib/probe-defaults.js` for details.
+configuration options. See [probe defaults](./docs/probe-defaults.md) for details.
```

Comment:
Since we stop mentioning `lib/prob-defaults.js` (as intended since it shouldn't be touched by the customer anymore), we should also update the later paragraph on line 75:
> Probe settings in the `appoptics-apm-config` configuration file will override those in `probe-defaults.js`, so the safest approach to changing an option is to add it to `appoptics-apm-config.json`.

To just something like "customize probe config in appoptics-apm-config" and not mention probe-defaults.js at all.

Reply:
Reworded in: 92f6f6766f9cc131e939536b023c0920cc9f34d6

Note: Any existing mention of `appopitics` is overwritten by the rebranding commit (assuming `git rebase` works as advertised, and if not, it will not arrive in the other repo. That way of doing thing is almost over. Almost.)